packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts

```
@@ -0,0 +1,1368 @@
+      const amounts = [BigNumber.from(100)];
+      const finalResultsUrl = VALID_URL;
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(ErrorRecipientCannotBeEmptyArray);
+    });
+
+    test('should throw an error if an amounts length to equal 0', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const recipients = [ethers.constants.AddressZero];
+      const amounts = [];
+      const finalResultsUrl = VALID_URL;
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(ErrorAmountsCannotBeEmptyArray);
+    });
+
+    test('should throw an error if an recipient and amounts not the same length', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const recipients = [ethers.constants.AddressZero];
+      const amounts = [
+        BigNumber.from(100),
+        BigNumber.from(100),
+        BigNumber.from(100),
+      ];
+      const finalResultsUrl = VALID_URL;
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(ErrorRecipientAndAmountsMustBeSameLength);
+    });
+
+    test('should throw an error if the recipients contain invalid addresses', async () => {
```

fix writing: 
'should throw an error if recipients contains invalid addresses'